### PR TITLE
edit README.md and fix leftover at gradient navbar

### LINKED ISSUE #49: Curl and Wget support
There is [PR](https://github.com/Webments/simplistika.css/pull/48) about it.

### PR DESCRIPTION
**Pull request by tansiret**
We have made this **together** as a free software project, fully open to contributors, not caring who they are. Why shall I not keep contributing to such a project?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,8 +31,9 @@
   </a>
 </p>
 
-simplistika.css is a lightweight, responsive and modern CSS framework.
+simplistika.css is a single-file, lightweight, responsive and modern CSS framework.
```
Comment (ardacebi) [inline README.md:34]: Is it worth referencing this feature in the first line of the framework README? That line is supposed to give information regarding the framework in general, is including an installation method as a feature on that line a decent option?

File: README.md [hunk 2]
```diff
@@ -31,8 +31,9 @@
...
 
+- Incredibly KISS.
```
Comment (ardacebi) [inline README.md:36]: What does KISS mean?

File: README.md [hunk 3]
```diff
@@ -48,7 +49,33 @@ simplistika.css is a lightweight, responsive and modern CSS framework.
 - [Copyright and License](#license)
 
 ## <a name="gettingstarted"></a>Getting started
-simplistika.css is available via either NPM (preferred), Yarn, or a CDN.
+simplistika.css is available via either direct installation, NPM, Yarn, or a CDN.
```
Comment (ardacebi) [inline README.md:52]: The direct installation option should be after the CDN option and NPM should remain as the preferred option as many developers still prefer to use package managers on their projects.

File: README.md [hunk 4]
```diff
@@ -48,7 +49,33 @@ simplistika.css is a lightweight, responsive and modern CSS framework.
...
+
+### KISSest Installation
+
+1. Clone the file wherever you want via wget or curl
+
+```shell
+wget https://raw.githubusercontent.com/Webments/simplistika.css/master/simplistika-min.css
```
Comment (ardacebi) [inline README.md]: `simplistika-min.css` should be `simplistika.min.css`

File: README.md [hunk 5]
```diff
@@ -48,7 +49,33 @@ simplistika.css is a lightweight, responsive and modern CSS framework.
...
+```
+
+or
+
+```shell
+curl -o simplistika.css "https://raw.githubusercontent.com/Webments/simplistika.css/master/simplistika-min.css"
```
Comment (ardacebi) [inline README.md]: `simplistika-min.css` should be `simplistika.min.css`

File: README.md [hunk 6]
```diff
@@ -48,7 +49,33 @@ simplistika.css is a lightweight, responsive and modern CSS framework.
...
+```
+
+2. Insert the line below to your head area in your HTML file
+
+```html
+<!-- index.html (that is just an example)-->
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="simplistika-min.css" />
```
Comment (ardacebi) [inline README.md:74]: `simplistika-min.css` should be `simplistika.min.css`

File: README.md [hunk 7]
```diff
@@ -60,7 +87,7 @@ npm i webments-simplistika
 2. Insert the line below to your head area in your HTML file
 
 ```html
-<!-- index.html -->
+<!-- index.html (that is just an example-->
```
Comment (ardacebi) [inline README.md]: A parenthesis is forgotten at the ending.

File: docs/components/navbars.html [hunk 8]
```diff
@@ -99,7 +99,7 @@ <h1>Navbars</h1>
             <li><a href="#">Item 3</a></li>
             <li><a href="#">Item 4</a></li>
             <input class="inp-text"></input>
-            <input type="checkbox" class="hm"></input>
+	    <span class="hm"></span>
```
Comment (ardacebi) [inline docs/components/navbars.html:102]: Please run a formatter before submitting any changes requested above. Automation using GitHub Actions will not be used on Webments repositories and it should be done locally before committing & pushing to remote.

**Review by ardacebi — CHANGES_REQUESTED** [review]
I've requested some changes and asked some questions regarding your edits.

File: README.md (hunk 2 above)
Comment (tansiret) [reply README.md:36]: https://acronyms.thefreedictionary.com/KISS means "Simple" in a nutshell. It is mostly used for very bare stuff, such as directly copying a file and using it.

File: README.md (hunk 3 above)
Comment (tansiret) [reply README.md:52]: Direct installation is a very KISS option. Directly getting the file with "wget" command is the easiest one among all. The preference is of the user.

File: README.md (hunk 4 above)
Comment (tansiret) [reply README.md]: k

File: README.md (hunk 5 above)
Comment (tansiret) [reply README.md]: k

File: README.md (hunk 2 above)
Comment (ardacebi) [reply README.md:36]: How is straightforward to replace KISS? Not sure if KISS is common.

File: README.md (hunk 6 above)
Comment (tansiret) [reply README.md:74]: k

File: README.md (hunk 2 above)
Comment (ardacebi) [reply README.md:36]: ...or just simple?

File: docs/components/navbars.html (hunk 8 above)
Comment (tansiret) [reply docs/components/navbars.html:102]: It just needs :retab, a format is useless.

File: README.md (hunk 3 above)
Comment (ardacebi) [reply README.md:52]: ok

File: docs/components/navbars.html (hunk 8 above)
Comment (ardacebi) [reply docs/components/navbars.html:102]: That wasn't a warning specifically for line 102, it was in general.

Comment (tansiret) [reply docs/components/navbars.html:102]: The thing above is recently a problem of Github.

File: README.md (hunk 1 above)
Comment (tansiret) [reply README.md:34]: That is needed. Its single file and that is a feature.

File: README.md (hunk 2 above)
Comment (tansiret) [reply README.md:36]: Web devs get it, its common for them.

File: README.md (hunk 1 above)
Comment (ardacebi) [reply README.md:34]: ok

File: README.md (hunk 2 above)
Comment (ardacebi) [reply README.md:36]: ok

File: docs/components/navbars.html (hunk 8 above)
Comment (ardacebi) [reply docs/components/navbars.html:102]: ok

**Review by ardacebi — APPROVED** [review]

**Comment by ardacebi** [comment]
@all-contributors please add @yutyo for documentation.

**Pull request merged**